index.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+require_once 'PlayerDao.php';
+
+$playerDao = new PlayerDao();
+
+$players = $playerDao->fetchAll();
+require_once 'index.html';
+?>
+    <body>
+    <nav class="navbar">
+        <ul class="links-container">
+            <li><a href="#collection">Collection</a></li>
+            <li><a href="#add-card-section">Add Card</a></li>
+
+        </ul>
+    </nav>
+    <h1>Match Attax</h1>
+
+    <section id="collection">
+<?php
+
+$html = '<div class="card-grid">';
+foreach ($players as $player) {
+    $html .= '<div class="player-card">' .
+        '<img src="'.$player->getImage().'" alt="A ' . $player->getName() . '">'
+        . '<p>Name: ' . $player->getName() . '</p>'
+        . '<p>Attack: ' . $player->getAttack() . '</p>'
+        . '<p>Defence: ' . $player->getDefence() . '</p>'
+        . '<p>Club: ' . $player->getClubId() . '</p>'
+        . '<p>Position: ' . $player->getPositionId() . '</p>'
+        . '</div>';
+}
+$html .= '</div>';
+
+echo $html;
```

Comment:
This would be really good to move into its own function, one that can be given the array of players, and will then return all of the HTML needed to display the cards.

This one would also need some unit tests

Reply:
Got it! 

Moved into own function in separate file and added 3 unit tests for it